pom.xml

```diff
@@ -109,24 +69,20 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+
     <dependencies>
         <dependency>
-            <groupId>org.broadleafcommerce</groupId>
-            <artifactId>broadleaf-third-party-integration-config</artifactId>
-            <version>default</version>
-            <scope>compile</scope>
-            <type>jar</type>
-        </dependency>
-        <dependency>
-            <groupId>org.broadleafcommerce</groupId>
-            <artifactId>broadleaf-common</artifactId>
-            <version>${blc.version}</version>
+            <groupId>com.broadleafcommerce.microservices</groupId>
+            <artifactId>broadleaf-payment-gateway-common</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
         </dependency>
+
         <dependency>
             <groupId>com.paypal.sdk</groupId>
             <artifactId>rest-api-sdk</artifactId>
             <version>1.14.0</version>
         </dependency>
+
```

Comment:
Remove the servlet API dependency version. It is already defined in the parent. It is possible that you will get an error about not having a version here. I can't remember when exactly the groupId/artifactId coordinates changed, but you might have to change the groupId/artifactId coordinates to work with servlet 3.0+